.github/workflows/code-examples.yml

```diff
@@ -445,7 +447,7 @@ jobs:
         if: steps.path-check.outputs.n-a != 'true'
         run: |
           mkdir -p ${{ env.TMP_FRAMEWORK_EXAMPLES_PATH }}
-          rsync -av --progress ${{ env.FRAMEWORK_EXAMPLES_PATH }}/ ${{ env.TMP_FRAMEWORK_EXAMPLES_PATH }} --include=/{,'[!node_modules]*/'{,dist/{,'**'}}} --exclude='*'
+          rsync -av --progress ${{ env.FRAMEWORK_EXAMPLES_PATH }}/ ${{ env.TMP_FRAMEWORK_EXAMPLES_PATH }} --exclude='node_modules' --include=/{,'*/'{,dist/{,'**'}}} --exclude='*'
```

Comment:
Was this the actual problem for Case 1 and 2 in the issue? Can you explain what's going on here?